### DISCUSSION TITLE
Change GitHub Payout Currency from NGN to USD

### DISCUSSION THREAD

**Discussion by Rakielle** [community/community#69539] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I linked a USD-denominated account for my GitHub Sponsors payouts, but Naira (NGN) is being displayed on Stripe. I'll prefer to receive the payouts in USD.
How can I change the payouts currency from Nigerian Naira (NGN) to US Dollars (USD)?

**Comment by Vijaykv5** [comment]
Hi @Rakielle  To change the payout currency for your GitHub Sponsors payouts from Nigerian Naira (NGN) to US Dollars (USD), you will need to contact GitHub Support. They can assist you in making this change, as it typically involves adjustments on their end and coordination with their payment processing partner, which may be Stripe.

Use the support contact form to submit a request explaining that you would like to change the currency of your payouts from NGN to USD.
GitHub Support will review your request and guide you through the process of changing the currency for your payouts.

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
